### PR TITLE
add broken test for #700

### PR DESCRIPTION
**Pull request by jw3126**
See #700. This is not a fix, just test cases.

### CODE REVIEW

File: test/layers/conv.jl [hunk 1]
```diff
@@ -33,3 +33,29 @@ end
...
+
+  l = Conv((3,3), 1=>1)
+  expected = fill(zero(eltype(l.weight)),5,5,1,1)
+  expected[2:end-1,2:end-1,1,1] = l.weight
+  @test expected == l(data)
+
+  l = Conv((3,1), 1=>1)
+  expected = fill(zero(eltype(l.weight)),5,7,1,1)
+  expected[2:end-1,4,1,1] = l.weight
+  @test_broken expected == l(data)
```
Comment (tejank10) [inline test/layers/conv.jl]: Using `@test_broken` is fine for now. Once [#99](https://github.com/FluxML/NNlib.jl/pull/99) gets merged, we would have to change it to `@test`.

Comment (jw3126) [reply test/layers/conv.jl]: So should I change it or not? Is the plan is to merge this PR first or not?

Comment (dhairyagandhi96) [reply test/layers/conv.jl]: Could change it to honor non square sizes now

Comment (jw3126) [reply test/layers/conv.jl]: Done!

**Comment by tejank10** [comment]
Thanks @jw3126 , good work!

**Pull request merged**